startup.sh

```diff
@@ -23,21 +23,30 @@
 
 ######## Environment check:
 
-# check for java home
-JAVA_EXE=$JAVA_HOME/bin/java
+# SET JAVA_HOME
+#JAVA_HOME=/usr/lib/jvm/java-14-openjdk-amd64/
+
+# check if java home exists
 
 if [ -z $JAVA_HOME ]; then
-  echo "JAVA_HOME not found. Please set it up properly."
-  JAVA_EXE=java
+  echo "Error - JAVA_HOME not found. Please set it up properly."
```

Comment:
I wonder if this old dance of `JAVA_HOME` env var is still needed. After all we just use it to locate the `java` executable, no? In most (all?) Linux distributions, if you install a JRE, `java` will be in PATH anyway - and `JAVA_HOME` is often not set (it's certainly not set in Arch Linux).  
So maybe this check can be removed?

Reply:
You might be right. I guess we can have linux systems with more than one JRE installed (sun and java), so we need to give the user the possibility of choosing which JAVA binary to use. I will try to work on the script to better clarify the choices available and to set-up JAVA_BIN automatically.